jac-uk/apply
# GDPR check

## What's included?
An additional page - akin to the eligibility check prior to applying to an exercise - with a GDPR statement and agreement.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Go to apply to any exercise.
- Go past eligibility check.
- Observe GDPR page.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work
🟠 Medium risk - this does change code that is shared with other areas
🔴 High risk - this includes a lot of changes to shared code

## Additional context
![image](https://user-images.githubusercontent.com/44227249/164195778-ef280e27-7b8c-48b9-80d6-6b1ab165df13.png)
![image](https://user-images.githubusercontent.com/44227249/164195829-a8909720-98e9-421a-8f0d-cc72323711a9.png)


---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
